<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QMainWindow" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLCC - Debug Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_state_initial">
       <property name="text">
        <string>Initial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_state_mainMenu">
       <property name="text">
        <string>Main Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_state_mapLoaded">
       <property name="text">
        <string>Playing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_state_countdown">
       <property name="text">
        <string>Countdown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_state_go">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_state_goalscored">
       <property name="text">
        <string>Goal scored!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>emit RLDetectedStates</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
